### PR TITLE
💄 Docs NavBar consistency and docs UI rework 

### PR DESCRIPTION
**Pull request by joshbermanssw**
As per #2157 

We wanted to make the navbar consistent across [tina.io](https://tina.io) site. To do so we have replaced the old navigation bar on [Tina docs](https://tina.io/docs) with the new one used elsewhere on the site 

test passed by jeoffrey and @bettybondoc

**Changes**
- added a sticky? prop to the `<NavBar/>` component so that we dont use sticky on the /docs (and sticky prop to `<Layout>`
- implemented `<Layout/>` into the `<DocumentNavigation/>` component 
- Re-worked the UI of the new docs to make the experience more seamless

Screenshot 2024-09-11 at 7 23 37 PM

**Figure: New navbar on /docs**

### CODE REVIEW

**Comment by isaaclombardssw** [comment]
Doesn't overlap quite right with the cookies pop-up.

**Comment by isaaclombardssw** [comment]
The close menu button is also obscuring some content, and that content should be removed as well.

**Comment by isaaclombardssw** [comment]
Legendary PBI. Looks really nice.

**Comment by isaaclombardssw** [comment]
Nitpicky, but the highlight is starting to escape.

**Review by Marxoz — APPROVED** [review]

**Review by Marxoz — COMMENTED** [review]
Need to address Isaac's changes first

**Comment by isaaclombardssw** [comment]
A couple more changes we could make to improve this. These came from the daily and could be a new PBI if you'd rather divide up the work.

Update the search to match the new ToC?


Create an alternative to the default dropdown menu.

**Comment by isaaclombardssw** [comment]
Another niche one – the tables don't really match the changed background. One for the tribute imo (or my solution would be to add a little border). 

Screenshot 2024-09-12 at 2 47 36 pm

**Comment by bettybondoc** [comment]
> Another niche one – the tables don't really match the changed background. One for a PBI or the tribute imo (or my solution for now would be to add a little border).
> 
> Screenshot 2024-09-12 at 2 47 36 pm

@isaaclombardssw - quicker solution would be to give the white/grey rows lower opacity so it gets a tint of the blue background

(I wasn't here 👀)

**Comment by joshbermanssw** [comment]
Screenshot 2024-09-13 at 2 30 44 PM

**Figure: Modernized version selector**

**Comment by isaaclombardssw** [comment]
Looks dope, just one more little styling issue to fix.

File: components/search/styles.tsx [hunk 1]
```diff
@@ -140,12 +140,17 @@ export const HitsWrapper = styled.div<HitsWrapperProps>`
...
...9' fill='white' /%3E%3Cmask id='mask0_566_318' style='mask-type:alpha' maskUnits='userSpaceOnUse' x='0' y='0' width='194' height='109'%3E%3Crect width='194' height='109' fill='url(%23paint0_linear_566_318)' /%3E%3C/mask%3E%3Cg mask='url(%23mask0_566_318)'%3E%3Crect width='194' height='109' fill='url(%23paint1_linear_566_318)' /%3E%3C/g%3E%3C/g%3E%3Cdefs%3E%3ClinearGradient id='paint0_linear_566_318' x1='97' y1='0' x2='97' y2='109' gradientUnits='userSpaceOnUse'%3E%3Cstop stop-color='%23D9D9D9' stop-opacity='0.45' /%3E%3Cstop offset='0.229052' stop-color='%23D9D9D9' stop-opacity='0.1678' /%3E%3Cstop offset='0.677779' stop-color='%23D9D9D9' stop-opacity='0.0513' /%3E%3Cstop offset='1' stop-color='%23D9D9D9' stop-opacity='0' /%3E%3C/linearGradient%3E%3ClinearGradient id='paint1_linear_566_318' x1='0' y1='54.5' x2='194' y2='54.5' gradientUnits='userSpaceOnUse'%3E%3Cstop stop-color='%2353E9DD' /%3E%3Cstop offset='0.34375' stop-color='%2368D7E4' /%3E%3Cstop offset='0.59375' stop-color='%2359BFF2' /%3E%3Cstop offset='1' stop-color='%234BA8FF' /%3E%3C/linearGradient%3E%3CclipPath id='clip0_566_318'%3E%3Crect width='194' height='109' fill='white' /%3E%3C/clipPath%3E%3C/defs%3E%3C/svg%3E ");
```
Comment (isaaclombardssw) [inline components/search/styles.tsx:152]: TODO mark for this one as well I reckon

**Review by isaaclombardssw — APPROVED** [review]
Suggestions only :)

**Pull request merged**